xwiki-platform-core/xwiki-platform-annotations/xwiki-platform-annotation-ui/src/main/resources/AnnotationCode/Translations.xml

```diff
@@ -73,27 +73,27 @@ annotations.action.create.submit.text=Add annotation
 annotations.action.create.cancel.text=Cancel
 annotations.action.create.selection.invalid=Please select a non-empty text in the page content.
 annotations.action.create.form.loaderror=Failed: 
-annotations.action.create.success=Annotation has been successfully added
+annotations.action.create.success=Annotation added!
```

Comment:
Same. Please use a comma instead of an exclamation.

Reply:
Done.